src/overlays/actors/ovl_En_Slime/z_en_slime.h

```diff
@@ -10,7 +14,34 @@ typedef void (*EnSlimeActionFunc)(struct EnSlime*, PlayState*);
 typedef struct EnSlime {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnSlimeActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0xC0];
-} EnSlime; // size = 0x208
+    /* 0x148 */ u8 iceBlockTimer;
+    /* 0x149 */ u8 eyeTexIndex;
+    /* 0x14A */ u8 damageEffectType;
+    /* 0x14C */ s16 timer;
+    /* 0x14E */ s16 idleRotY;
+    /* 0x150 */ s16 respawnRotY;
+    /* 0x152 */ s16 respawnTime;
+    /* 0x154 */ union { 
+                    Vec3f iceBlockSnapPos;
+                    Vec3f wobbleRot;
+                };
+    /* 0x160 */ void* dropObjectTex;
+    /* 0x164 */ f32 effectAlpha;
+    /* 0x168 */ f32 effectScale;
+    /* 0x16C */ f32 frozenSteamScale;
+    /* 0x170 */ f32 distLimit;
+    /* 0x174 */ Vec3f respawnScale;
+    /* 0x180 */ Vec3f limbPos[EN_SLIME_LIMBPOS_COUNT];
+    /* 0x1BC */ ColliderCylinder collider;
+} EnSlime; /* size = 0x208 */
```

Comment:
```suggestion
} EnSlime; // size = 0x208
```
Just a styling thing